conditional/blueprints/intro_evals.py

```diff
@@ -32,10 +32,24 @@ def display_intro_evals(internal=False, user_dict=None):
     log.info('Display Intro Evals Listing')
 
     # get user data
-    def get_fid_cm_count(member_id):
-        return len([a for a in FreshmanCommitteeAttendance.query.filter(
-            FreshmanCommitteeAttendance.fid == member_id)
-            if CommitteeMeeting.query.filter(CommitteeMeeting.id == a.meeting_id).first().approved])
+    def get_fid_cm(member):
+        c_meetings = [{
+            "uid": cm[0],
```

Comment:
fid? and why do you need this for every entry, isn't the function called with the fid as an argument?

Reply:
These are merged with the entries from those with accounts. Although I don't believe that it is actively used, in the event that somebody wants to it will be much easier if it is the same key as the member entries.